content/en/docs/contribute/localization.md

```diff
@@ -21,7 +22,7 @@ This page shows you how to [localize](https://blog.mozilla.org/l10n/2011/12/14/i
 
 ## Getting started
 
-Because contributors can't approve their own pull requests, you need at least two contributors to begin a localization. 
+Because contributors can't approve their own pull requests, you need at least two contributors to begin a localization.
 
 All localization teams must be self-sustaining with their own resources. We're happy to host your work, but we can't translate it for you.
```

Comment:
@celestehorgan an aside: you might want to reword this sentence too, based on https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you

(it's fine to save that for a different pull request)